pymatgen/io/lobster/inputs.py

```diff
@@ -54,25 +54,24 @@
 
 
 class Lobsterin(UserDict, MSONable):
-    """
-    This class can handle and generate lobsterin files
-    Furthermore, it can also modify INCAR files for lobster, generate KPOINT files for fatband calculations in Lobster,
+    """Handle and generate lobsterin files.
+    Furthermore, it can also modify INCAR files for LOBSTER, generate KPOINTS files for fatband calculations in Lobster,
     and generate the standard primitive cells in a POSCAR file that are needed for the fatband calculations.
     There are also several standard lobsterin files that can be easily generated.
-    """
 
-    # reminder: lobster is not case sensitive
+    Reminder: LOBSTER is not case sensitive.
+    """
 
-    # keyword + one float can be used in file
-    FLOAT_KEYWORDS = (
+    # Keyword + float
+    _FLOAT_KEYWORDS: tuple[str, ...] = (
         "COHPstartEnergy",
         "COHPendEnergy",
         "gaussianSmearingWidth",
         "useDecimalPlaces",
         "COHPSteps",
     )
-    # one of these keywords +endstring can be used in file
-    STRING_KEYWORDS = (
+    # One of these keywords + string
```

Comment:
is the "can be used in file ..." missing here?

Reply:
Yeps... But this comment and its siblings are a bit ambiguous to me?

https://github.com/materialsproject/pymatgen/blob/2982968a44d0ff65392057488aee4871210bc847/pymatgen/io/lobster/inputs.py#L66
https://github.com/materialsproject/pymatgen/blob/2982968a44d0ff65392057488aee4871210bc847/pymatgen/io/lobster/inputs.py#L74
https://github.com/materialsproject/pymatgen/blob/2982968a44d0ff65392057488aee4871210bc847/pymatgen/io/lobster/inputs.py#L85

Especially the last one: https://github.com/materialsproject/pymatgen/blob/2982968a44d0ff65392057488aee4871210bc847/pymatgen/io/lobster/inputs.py#L119

Reply:
yup, I think these commands can be summarized/fused and then be put into the docstring, where it's easier to look up the usage.

Reply:
> yup, I think these commands can be summarized/fused and then be put into the docstring, where it's easier to look up the usage.

I think this sounds like a task for the LOBSTER manual, not `pymatgen`? Perhaps we should somehow clarify these expressions here (or maybe include an example for each?)

Reply:
ah now that I read it again, I think "+ float" and "+ string" is understandable.

Reply:
I actually don't think the commments should be removed. They refer to the way that we later on store the information in pymatgen. This has not much to do with the Lobster manual itself as the Lobster manual does not introduce any classification of their commands.

Reply:
Yeah, that's what I realized too when I was going through it again, see above (could have phrased that more clearly)

Reply:
Thanks for the helpful discussion!